### PR TITLE
Yocto 3.1 Recently added fixes

### PR DESCRIPTION
**Pull request by Rajakumaran1106**

### CODE REVIEW

File: recipes-support/wrp-c/files/replace-strncpy-by-memcpy.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/src/wrp-c.c b/src/wrp-c.c
+index 5a2ed43..b729e5e 100644
+--- a/src/wrp-c.c
++++ b/src/wrp-c.c
+@@ -716,7 +716,7 @@ static ssize_t __wrp_keep_alive_to_string (char **bytes )
+         data = ( char* ) malloc( sizeof( char ) * ( length + 1 ) );   /* +1 for '\0' */
+ 
+         if( NULL != data ) {
+-            strncpy( data, keep_alive_fmt, length );
++            memcpy( data, keep_alive_fmt, length );
```
Comment (manigandanj) [inline recipes-support/wrp-c/files/replace-strncpy-by-memcpy.patch:10]: What is the need for this change?

File: recipes-common/rtmessage/rtmessage_git.bbappend [hunk 2]
```diff
@@ -0,0 +1 @@
+DEPENDS += "rdk-logger"
```
Comment (manigandanj) [inline recipes-common/rtmessage/rtmessage_git.bbappend:1]: Is this possible to change commit message?

File: recipes-ccsp/ccsp/ccsp-cr.bbappend [hunk 3]
```diff
@@ -9,3 +9,6 @@ do_install_append() {
     # Config files and scripts
     install -m 644 ${WORKDIR}/cr-deviceprofile_turris.xml ${D}/usr/ccsp/cr-deviceprofile.xml
 }
+
+LDFLAGS += "-lbreakpadwrapper"
```
Comment (manigandanj) [inline recipes-ccsp/ccsp/ccsp-cr.bbappend:13]: Is this possible to change commit message?

File: recipes-support/wrp-c/files/replace-strncpy-by-memcpy.patch (hunk 1 above)
Comment (Rajakumaran1106) [reply recipes-support/wrp-c/files/replace-strncpy-by-memcpy.patch:10]: It throws an error, with that particular wrp-c version, fixed in latest version,

File: recipes-common/rtmessage/rtmessage_git.bbappend (hunk 2 above)
Comment (Rajakumaran1106) [reply recipes-common/rtmessage/rtmessage_git.bbappend:1]: Could not change the commit alone, if we want to , we  need  to push the new PR

File: recipes-ccsp/ccsp/ccsp-cr.bbappend (hunk 3 above)
Comment (Rajakumaran1106) [reply recipes-ccsp/ccsp/ccsp-cr.bbappend:13]: Could not change the commit alone, if we want to , we need to push the new PR

**Comment by manigandanj** [comment]
Raised corrective pull request https://github.com/rdkcentral/meta-turris/pull/160.

**Pull request closed without merging**